Sources/SwiftDocC/Model/Rendering/DocumentationContentRenderer.swift

```diff
@@ -230,7 +230,7 @@ public class DocumentationContentRenderer {
             }
 
             // Verify that the current platform is in beta and the version number matches the introduced platform version.
-            guard current.beta && SemanticVersion(introduced).isEqualToVersionTriplet(current.version) else {
+            guard current.beta && VersionTriplet(introduced) >= current.version else {
```

Comment:
Minor comment: I was trying to work towards only using `SemanticVersion` as the type used internally by DocC for semantic versioning, if we could head in that direction that would be helpful (context: https://github.com/swiftlang/swift-docc/issues/970)

Reply:
+1 to this. Can we use Semantic Version here?

Reply:
I changed this code to use `VersionTriplet` since that conforms to `Comparable` making functions such as `isEqualToVersionTriplet` unnecessary. If we're moving towards using `SemanticVersion` then I can add `Comparable` conformance to that and refactor this code.